WordPressVIPMinimum/Tests/SVG/HTMLCodeSniff.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Unit test class for WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Tests\SVG;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+
+/**
+ * Unit test class for PHP opening tags in SVG files.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+class StrippingTagsUnitTest extends AbstractSniffUnitTest {
+
+	/**
+	 * Returns the lines where errors should occur.
+	 *
+	 * @return array <int line number> => <int number of errors>
+	 */
+	public function getErrorList() {
+		return array();
+	}
+
+	/**
+	 * Returns the lines where warnings should occur.
+	 *
+	 * @return array <int line number> => <int number of warnings>
+	 */
+	public function getWarningList() {
+		return array(
+			7 => 1,
```

Comment:
Where's the test file that this relates to? I don't see it in this PR.

Reply:
Has been added now. :-)